dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -0,0 +1,450 @@
+                .withContentType(contentGenericType, "$!{body}")
+                .friendlyName(nameTitle)
+                .title(nameTitle)
+                .nextPersisted();
+
+        PublishFactory.publishAsset(container, systemUser, false, false);
+        template = new TemplateDataGen().withContainer(container.getIdentifier()).nextPersisted();
+
+        folder = new FolderDataGen().site(site).nextPersisted();
+
+        PublishFactory.publishAsset(template, systemUser, false, false);
+
+        //Create Contentlet in English
+        final Contentlet contentlet1 = new ContentletDataGen(contentGenericId)
+                .languageId(defaultLang)
+                .folder(folder)
+                .host(site)
+                .setProperty("title", "content1")
+                .setProperty("body", "content1")
+                .nextPersisted();
+
+        contentlet1.setIndexPolicy(IndexPolicy.WAIT_FOR);
+        contentlet1.setIndexPolicyDependencies(IndexPolicy.WAIT_FOR);
+        contentlet1.setBoolProperty(Contentlet.IS_TEST_MODE, true);
+        contentletAPI.publish(contentlet1, systemUser, false);
+
+        final String pageName = "our-page";
+
+        final HTMLPageAsset pageEnglishVersion = new HTMLPageDataGen(folder, template).languageId(1)
+                .pageURL(pageName)
+                .friendlyName(pageName)
+                .title(pageName).nextPersisted();
+        final String uuid = UUIDGenerator.generateUuid();
+        final MultiTree multiTree = new MultiTree(pageEnglishVersion.getIdentifier(), container.getIdentifier(), contentlet1.getIdentifier() ,uuid,0);
+        APILocator.getMultiTreeAPI().saveMultiTree(multiTree);
+
+        HTMLPageDataGen.publish(pageEnglishVersion);
+
+    }
+
+    @AfterClass
+    public static void drop(){
+    }
+
+    @Test
+    public void Test_Non_Incremental_Create_Default_Index_Run_Non_Incrementally_Expect_New_Index_Keep_Alias_And_Default()
+            throws DotPublishingException, JobExecutionException, DotDataException, IOException, DotSecurityException {
+
+        final List<String> indicesBeforeTest = siteSearchAPI.listIndices();
+        for(final String index:indicesBeforeTest) {
+            esIndexAPI.delete(index);
+        }
+        final String jobId = UUIDUtil.uuid();
+        final String alias = "any-alias-"+System.currentTimeMillis();
+        final JobDataMap jobDataMap = new JobDataMap();
+        jobDataMap.put(SiteSearchJobImpl.RUN_NOW, Boolean.TRUE.toString());
+        //Even though we are requesting it to be incremental it'll run as non incremental because it is a run now type of job.
+        jobDataMap.put(SiteSearchJobImpl.INCREMENTAL, Boolean.TRUE.toString());
+        jobDataMap.put(SiteSearchJobImpl.INDEX_ALIAS, alias);
+        jobDataMap.put(SiteSearchJobImpl.JOB_ID, jobId);
+        jobDataMap.put(SiteSearchJobImpl.QUARTZ_JOB_NAME, SiteSearchJobImpl.RUNNING_ONCE_JOB_NAME);
+        jobDataMap.put(SiteSearchJobImpl.INCLUDE_EXCLUDE, "all");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "all" appears 4 times in this file; the first occurrence is on line 165](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093862)